### PR TITLE
Create Dispute Api Project

### PR DESCRIPTION
**Pull request by rantifamilusi**
# Description

This PR includes the following proposed change(s):

- https://justice.gov.bc.ca/jira/browse/TCVP-42

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring / Documentation
- [ ] Version change

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Does the change impact or break the Docker build?

- [ ] Yes
- [x] No

If Yes: Has Docker been updated accordingly?

- [ ] Yes
- [ ] No

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by pbolduc** [comment]
This PR does not meet our code coverage targets. 41.01%

image

**Comment by rantifamilusi** [comment]
Updated now

image

File: src/backend/TrafficCourtsApi/DisputeApi.Web.Test/Features/Tickets/Models/TicketModelTest.cs [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+namespace DisputeApi.Web.Test.Features.Tickets.Models
+{
+    public class TicketModelTest
+    {
+
+        [Test]
+        public void can_create_class()
+        {
+            var ticket = new Ticket { Id = "Id", Description ="Description" };
+            Assert.IsNotNull(ticket);
```
Comment (pbolduc) [inline src/backend/TrafficCourtsApi/DisputeApi.Web.Test/Features/Tickets/Models/TicketModelTest.cs:13]: nitpick, the new operator will never return null. Either the object will be created or an exception would be thrown.

**Review by pbolduc — APPROVED** [review]

File: src/backend/BcGov.TrafficCourts.Api/DisputeApi.Web.Test/Features/Tickets/Services/TicketServiceTest.cs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DisputeApi.Web.Test.Features.Tickets.Services
+{
+    public class TicketServiceTest
+    {
+        private TicketService _sut;
+          private readonly Mock<ILogger<TicketService>> _loggerMock = LoggerServiceMock.LoggerMock<TicketService>();
```
[the remark this answers is not in the document]
Comment (rantifamilusi) [reply src/backend/BcGov.TrafficCourts.Api/DisputeApi.Web.Test/Features/Tickets/Services/TicketServiceTest.cs]: What are the errors that it caused. We used it FAMS and did not experience issues. Maybe we can change ?

**Pull request merged**